.goreleaser.yml

```diff
@@ -0,0 +1,356 @@
+    main: ./cmd/executor/main.go
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    goos:
+      - linux
+    goarch:
+      - amd64
+  - env: [CGO_ENABLED=0]
+    id: binoculars
+    binary: binoculars
+    main: ./cmd/binoculars/main.go
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    goos:
+      - linux
+    goarch:
+      - amd64
+  - env: [CGO_ENABLED=0]
+    id: lookout
+    binary: lookout
+    main: ./cmd/lookout/main.go
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    goos:
+      - linux
+    goarch:
+      - amd64
+  - env: [CGO_ENABLED=0]
+    id: lookoutingester
+    binary: lookoutingester
+    main: ./cmd/lookoutingester/main.go
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    goos:
+      - linux
+    goarch:
+      - amd64
+  - env: [CGO_ENABLED=0]
+    id: eventingester
+    binary: eventingester
+    main: ./cmd/eventingester/main.go
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    goos:
+      - linux
+    goarch:
+      - amd64
+  - env: [CGO_ENABLED=0]
+    id: testsuite
+    binary: testsuite
+    main: ./cmd/testsuite/main.go
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    goos:
+      - linux
+    goarch:
+      - amd64
+  - env: [CGO_ENABLED=0]
+    id: armadactl
+    binary: armadactl
+    main: ./cmd/armadactl/main.go
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    goos:
+      - windows
+      - darwin
+      - linux
+    goarch:
+      - amd64
+      - arm64
+
+archives:
+  - id: armadactl
+    builds:
+      - armadactl
+    allow_different_binary_count: true
+    name_template: "armadactl_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
+    format_overrides:
+      - goos: windows
+        format: zip
+
+# macOS Universal Binaries-*
+universal_binaries:
+  - replace: true
+    id: armadactl
+    name_template: 'armadactl'
+
+signs:
+  - artifacts: checksum
+
+sboms:
+  - artifacts: archive
+
+dockers:
+  -
+    goos: linux
+    goarch: amd64
+    image_templates:
+      - "{{ .Env.DOCKER_REPO }}/armada:{{ .Major }}"
+      - "{{ .Env.DOCKER_REPO }}/armada:{{ .Major }}.{{ .Minor }}"
+      - "{{ .Env.DOCKER_REPO }}/armada:{{ .Major }}.{{ .Minor }}.{{ .Patch }}"
+      - "{{ .Env.DOCKER_REPO }}/armada:latest"
+    build_flag_templates:
+      - --label=org.opencontainers.image.title=armada
+      - --label=org.opencontainers.image.description="Armada Bundle"
+      - --label=org.opencontainers.image.url=https://hub.docker.com/r/gresearchdev/armada
+      - --label=org.opencontainers.image.source=https://github.com/G-Research/armada
+      - --label=org.opencontainers.image.version={{ .Version }}
+      - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
+      - --label=org.opencontainers.image.revision={{ .FullCommit }}
+      - --label=org.opencontainers.image.base.name=alpine:3.17.0
+      - --label=org.opencontainers.image.base.digest=c0d488a800e4
+      - --label=org.opencontainers.image.licenses=Apache-2.0
+      - --label=org.opencontainers.image.vendor=G-Research
+    extra_files:
```

Comment:
Is this right? I'd have expected only 1 config.

Reply:
yep, the bundled, I renamed armada -> server, and use armada for the bundled image